<ui version="4.0" >
 <class>FillAndStroke</class>
 <widget class="QWidget" name="FillAndStroke" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>185</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <widget class="QTabWidget" name="Fill" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="_fillPage" >
      <attribute name="title" >
       <string>Fill</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Color:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="Kst::ColorButton" name="_color" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Style:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QComboBox" name="_fillStyle" />
       </item>
       <item row="2" column="0" colspan="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_stroke" >
      <attribute name="title" >
       <string>Stroke</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Style:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QComboBox" name="_strokeStyle" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QDoubleSpinBox" name="_width" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Brush Color:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="Kst::ColorButton" name="_brushColor" />
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Brush Style:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QComboBox" name="_brushStyle" />
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Join Style:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QComboBox" name="_joinStyle" />
       </item>
       <item row="5" column="0" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>Cap Style:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <widget class="QComboBox" name="_capStyle" />
       </item>
       <item row="6" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Kst::ColorButton</class>
   <extends>QToolButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <includes/>
 <resources/>
 <connections/>
</ui>
